book/style_guide.md

```diff
@@ -180,6 +180,141 @@ let selectedProfile = (
 })
 ```
 
+## Naming Convention
+
+### Abbreviations and Acronyms
+
+**It's recommended** to use full concise words over abbreviations and acronyms, unless they are well-known and/or
+commonly used.
+
+Correct:
+
+```nu
+query-user --id 123
+
+$user.name | str downcase
+```
+
+Incorrect:
+
+```nu
+qry-usr --id 123
+
+$user.name | string downcase
+```
+
+### Case
+
+#### Commands
+
+**It's recommended to** use kebab-case for command names with multiple words.
+
+Correct:
+
+```nu
+fetch-user --id 123
+```
+
+Incorrect:
+
+```nu
+fetch_user --id 123
+fetchUser --id 123
+```
+
+See also [Naming Commands](custom_commands.md#naming-commands).
+
+#### Sub-Commands
```

Comment:
I *think* we could combine the "Commands" and "Subcommands" sections and say something like:

> **It's recommended to** use kebab-case for command and subcommand names with multiple words.
>
> Subcommands are commands that are logically grouped under a parent command and separated from the command name by a space.